### PR TITLE
Style component global cleanup

### PR DESCRIPTION
**Pull request by Tbaut**
- Not touching https://github.com/polkadot-js/extension/blob/bc4e57e353940dbabb8a53ae7dc63e7038b924ea/packages/extension-ui/src/components/TextInputs.ts since I would need to duplicate the styling, I don't think it would make sense.

### CODE REVIEW

File: packages/extension-ui/src/Popup/Export.tsx [hunk 1]
```diff
@@ -58,10 +62,15 @@ function Export ({ match: { params: { address } } }: Props): React.ReactElement<
...
         text={t<string>('Export account')}
       />
-      <div>
+      <div className={className}>
         <Address address={address}>
-          <MovedWarning danger>{t<string>("You are exporting your account. Keep it safe and don't share it with anyone.")}</MovedWarning>
-          <ActionArea>
+          <Warning
+            className='movedWarning'
+            isDanger
```
Comment (Tbaut) [inline packages/extension-ui/src/Popup/Export.tsx:69]: That was a bug `danger` -> `isDanger`

Comment (jacogr) [reply packages/extension-ui/src/Popup/Export.tsx:69]: Cool! Love the drive-by bugs being fixed.

File: packages/extension-ui/src/Popup/Derive/AddressDropdown.tsx [hunk 2]
```diff
@@ -95,7 +78,31 @@ export default styled(AddressDropdown)(({ theme }: ThemeProps) => `
...
+  }
+
+  .dropdown {
+    position: absolute;
     visibility: hidden;
+    width: 510px;
+    z-index: 100;
+    background: ${theme.bodyColor};
+    max-height: 0;
+    overflow: auto;
```
Comment (Tbaut) [inline packages/extension-ui/src/Popup/Derive/AddressDropdown.tsx:92]: this used to be a scroll with a useless disabled horizontal scrollbar.

File: packages/extension-ui/src/components/ButtonArea.tsx [hunk 3]
```diff
@@ -3,22 +3,32 @@
...
+export default styled(ButtonArea)(({ theme }: ThemeProps) => `
   display: flex;
   flex-direction: row;
   background: ${theme.highlightedAreaBackground};
   border-top: 1px solid ${theme.inputBorderColor};
   padding: 12px 24px;
+  margin-left: 0;
+  margin-right: 0;
 
-  &&& {
```
Comment (Tbaut) [inline packages/extension-ui/src/components/ButtonArea.tsx:16]: totally removed, it made no change from what I could see (also I don't really understand what it should do)

File: packages/extension-ui/src/components/MenuItem.tsx [hunk 4]
```diff
@@ -17,23 +17,27 @@ function MenuItem ({ children, className = '', title }: Props): React.ReactEleme
...
+export default styled(MenuItem)(({ theme }: ThemeProps) => `
   min-width: 13rem;
   padding: 0 16px;
   max-width: 100%;
 
   > .itemTitle {
-    margin: 0;
-  }
-
-  &+& {
```
Comment (Tbaut) [inline packages/extension-ui/src/components/MenuItem.tsx:36]: same here, I removed it entirely, made no difference I could see. This should apply when we have a `menuItem` directly nested into a `menuIdem`, but this is never the case (any more?) in practice.

**Comment by Tbaut** [comment]
Other than the couple comments I left, there should be nothing new/changed.

File: packages/extension-ui/src/Popup/Authorize/Request.tsx [hunk 5]
```diff
@@ -38,6 +38,8 @@ function Request ({ authId, className, isFirst, request: { origin }, url }: Prop
     [authId, onAction]
   );
 
+  console.log('isFirst', isFirst);
+
```
Comment (jacogr) [inline packages/extension-ui/src/Popup/Authorize/Request.tsx:42]:
```suggestion
```

File: packages/extension-ui/src/Popup/Authorize/Request.tsx [hunk 6]
```diff
@@ -60,6 +62,7 @@ function Request ({ authId, className, isFirst, request: { origin }, url }: Prop
         </div>
         {isFirst && (
           <>
+            {console.log('hoooo')}
```
Comment (jacogr) [inline packages/extension-ui/src/Popup/Authorize/Request.tsx:65]:
```suggestion
```

**Review by jacogr — APPROVED** [review]
Lgtm. Have not tested through, but really seems to fit the bill and make stuff maintainable again.

**Comment by Tbaut** [comment]
Thanks, I've been trying to test every step very carefully. Hope I didn't miss anything important.
I'll resolve conflicts and merge.

**Pull request merged**